server/sonar-web/src/main/js/components/navigator/filters/ajax-select-filters.js

```diff
@@ -453,38 +453,6 @@ const AssigneeFilterView = AjaxSelectFilterView.extend({
 });
 
 
-const ReporterFilterView = AjaxSelectFilterView.extend({
```

Comment:
Hum, no, in fact this commit already belongs to "SONAR-7473 Drop 'reporter' field from UI" commit.
@stas-vilchik So it should be ok ?